nestle.py

```diff
@@ -856,16 +947,23 @@ def prior_transform(u):
         if update_interval < 1:
             raise ValueError("update_interval must be >= 1")
 
+    # Parallel evaluation.
+    if n_threads is None or n_threads == 1:
+        n_threads = 1
+        pool = FakePool()
+    else:
+        if pool is None:
+            pool = concurrent.futures.ThreadPoolExecutor(n_threads)
```

Comment:
ThreadPoolExedcutor is almost never of any use. Why not drop n_threads and simply require the user to supply a pool if they want parallelization? You do need an n_queue, unfortunately, which should be at least the number of cores that will be working.